hoss/utilities.py

```diff
@@ -75,6 +84,24 @@ def get_opendap_nc4(
     return move_downloaded_nc4(output_dir, downloaded_nc4)
 
 
+def unexecuted_url_requested(mimetype: str) -> bool:
```

Comment:
I think the type needs to be updated.
```suggestion
def unexecuted_url_requested(mimetype: str | None) -> bool:
```